### ISSUE TITLE
showcases: ensure a minimum of words in alternative texts

### ISSUE THREAD

**Issue by mfranzke** [db-ui/mono#3035]
As NVDA screenreader would output "Unlabeled graphic, _Alternativetext_. To get missing image descriptions, open the context menu." (where _Alternativetext_ stands for the text that is inserted within an `alt`-attribute) in case that the alternative-text is too short or only one word, and NVDA seems to try to interpret the images meaning instead, fails and doesn't only fall back to that alternative-text, we should ensure our showcases and examples to contain enough contents (as well as mention that content production relevant aspect within our content authors guidelines).

DBUXDS-3210
